apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -0,0 +1,210 @@
+                                  >job.tar.gz</a
+                                >
+                              </td>
+                              <td>
+                                Contains all files necessary for grading; this is what is mounted to
+                                <code>/grade</code> when your job is run.
+                              </td>
+                            </tr>
+                            <tr>
+                              <td>
+                                <a
+                                  href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                                    .grading_job.id}/file/archive.tar.gz"
+                                  >archive.tar.gz</a
+                                >
+                              </td>
+                              <td>
+                                A snapshot of <code>/grade</code> after your job has been executed.
+                              </td>
+                            </tr>`
+                        : ''}
+                      <tr>
+                        <td>
+                          <a
+                            href="${resLocals.urlPrefix}/grading_job/${gradingJobQueryResult
+                              .grading_job.id}/file/results.json"
+                            >results.json</a
+                          >
+                        </td>
+                        <td>
+                          Contains the PrairieLearn-generated results, which includes the contents
+                          of your
+                          <code>results.json</code> as well as some PrairieLearn metadata.
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <a
+                            href="<%= urlPrefix %>/grading_job/<%= grading_job.id %>/file/output.log"
+                            >output.log</a
+                          >
+                        </td>
+                        <td>
+                          Contains the raw output from stdout/stderr for your job. Lines beginning
+                          with
+                          <code>container &gt;</code> are from your container; the rest are
+                          diagnostic logs from PrairieLearn.
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Job Output</div>
+            <div class="card-body">
+              ${gradingJobQueryResult.grading_job.s3_bucket &&
+              gradingJobQueryResult.grading_job.s3_root_key
+                ? html`
+                    <script>
+                      $.get('<%= urlPrefix %>/grading_job/<%= grading_job.id %>/file/output.log')
```

Comment:
`<%=` would have previously been templated by EJS, but won't be any more. Rather than having templates that spit out custom JS, it'd be better to refactor this so that this code can pick this URL off a data attribute. For instance, you could add `data-output-url` to the `<pre>` tag and read that here so you know where to fetch the output from. Let me know if you need clarification on that!

Reply:
Resolved in commit 5a8a71e.